File: solutions/java/darts/2/src/main/java/Darts.java
class Darts {
    int score(double xOfDart, double yOfDart) {
        
        int points = 0;
        double h  = Math.sqrt((xOfDart * xOfDart) + (yOfDart * yOfDart)); 

        
        
        if(h > 10) {
            points = 0;
            
        } else if(h > 5) {
            points = 1;
            
        } else if(h > 1) {
            points = 5;
            
        } else {
            points = 10;
        }
            
        return points;
    }
}

File: solutions/c/darts/2/darts.c
#include "darts.h"

uint8_t score(coordinate_t landing_position) {
    
    uint8_t points = 0;
    
    float h = hypot(landing_position.x, landing_position.y);
  
    if(h > 10.0F) {
        points = 0;
        
    } else if(h > 5.0F) {
        points = 1;
        
    } else if(h > 1.0F) {
        points = 5;
        
    } else{
        points = 10;
    }

    return points;
}